docker-mcp-pr7-fixes.md

```
@@ -0,0 +1,1334 @@
+> -            log_dir=os.getenv("LOG_DIR", str(get_data_dir() / "logs")),
+> -            log_level=os.getenv("LOG_LEVEL"),
+> -            max_file_size_mb=int(os.getenv("LOG_FILE_SIZE_MB", "10"))
+> -        )
+>---
+
+- [ ] [DIFF BLOCK - coderabbitai[bot] - docker_mcp/server.py:42-46]
+> -                # Update docker-compose.yaml with new port
+> -                sed -i.bak "s|\"8000:8000\"|\"${available_port}:8000\"|g" "$compose_file"
+> -                sed -i.bak "s|FASTMCP_PORT: \"8000\"|FASTMCP_PORT: \"8000\"|g" "$compose_file"
+> -                rm -f "${compose_file}.bak"
+> -                
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 16

(MD009, no-trailing-spaces)